### PR TITLE
Updated dead links to several pages

### PR DESCRIPTION
**Pull request by Pilargit12**
Updated broken links provided by Monica Ganbrois: 

Page | link title | link URL | Revised
-- | -- | -- | --
High availability database clusters | Licensed Database Products | https://docs.developer.gov.bc.ca/licensed-database-products/ | Removed
Reusable services list | BC Address Geocoder repository | https://github.com/bcgov/api-specs/blob/master/geocoder/glossary.md#outputSRS | Changed to: https://github.com/bcgov/api-specs/tree/master/geocoder
Install the oc command line tool | available from Redhat | https://docs.openshift.com/container-platform/4.12/web_console/odc-using-web-terminal.html#odc-access-web-terminal_odc-using-web-terminal | Changed to: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/web_console/web-terminal
Set up advanced metrics in Sysdig Monitor | import your application metrics endpoint into Sysdig metrics | https://docs.sysdig.com/en/docs/sysdig-monitor/monitoring-integrations/custom-integrations/collect-prometheus-metrics/migrating-from-promscrape-v1-to-v2/#migrate-using-default-configuration | Changed to: https://docs.sysdig.com/en/docs/sysdig-monitor/integrations/working-with-integrations/custom-integrations/collect-prometheus-metrics/migrating-from-promscrape-v1-to-v2/#migrating-from-promscrape-v1-to-v2
Set up advanced metrics in Sysdig Monitor | Migrate Using Default Configuration | https://docs.sysdig.com/en/docs/sysdig-monitor/monitoring-integrations/custom-integrations/collect-prometheus-metrics/migrating-from-promscrape-v1-to-v2/#migrate-using-default-configuration | Changed to: https://docs.sysdig.com/en/docs/sysdig-monitor/integrations/working-with-integrations/custom-integrations/collect-prometheus-metrics/migrating-from-promscrape-v1-to-v2/#migrate-using-default-configuration
Set up a User Defined Monitoring in OpenShift | Sysdig - Automatically scraping any Kubernetes pods | https://docs.sysdig.com/en/docs/sysdig-monitor/monitoring-integrations/custom-integrations/collect-prometheus-metrics/#agent-compatibility | Changed to: https://docs.sysdig.com/en/docs/sysdig-monitor/integrations/working-with-integrations/custom-integrations/collect-prometheus-metrics/#agent-compatibility
BC Government OpenShift DevOps Security Considerations | https://www2.gov.bc.ca/gov/content/governments/services-for-government/bc-bid-resources/goods-and-services-catalogue/im-it-security-services | https://www2.gov.bc.ca/gov/content/governments/services-for-government/bc-bid-resources/goods-and-services-catalogue/im-it-security-services | Changed to: https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security
External training resources | Building Secure and Reliable Systems | https://static.googleusercontent.com/media/sre.google/en//static/pdf/building_secure_and_reliable_systems.pdf | Changed to: https://sre.google/books/building-secure-reliable-systems/#:~:text=Security%20is%20crucial%20to%20the,systems%20that%20are%20fundamentally%20secure.
Argo CD usage | https://kustomize.io/tutorial | https://kustomize.io/tutorial | Changed to: https://kustomize.io/
About the design system | Joint Working Group (2 times on the page) | https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/digital-delivery/web-property-process/web-property-applications | Changed to: https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/digital-delivery/web-property-process
Deploy an Application | official OpenShift documentation | https://docs.openshift.com/container-platform/latest/applications/application_life_cycle_management/creating-applications-using-cli.html | Changed to: https://docs.openshift.com/container-platform/4.8/applications/creating_applications/creating-applications-using-cli.html
Maintain an application | Check application health after outage | https://docs.developer.gov.bc.ca/maintain-an-application/check-application-health-after-outage/ | Changed to: https://docs.developer.gov.bc.ca/check-application-health-after-outage/
Maintain an application | Building secure and reliable systems e-book | https://static.googleusercontent.com/media/sre.google/en//static/pdf/building_secure_and_reliable_systems.pdf | Changed to: https://sre.google/books/building-secure-reliable-systems/

Page	link title	link URL	Revised 
High availability database clusters	 Licensed Database Products	https://docs.developer.gov.bc.ca/licensed-database-products/	 Removed 
Reusable services list	BC Address Geocoder repository	https://github.com/bcgov/api-specs/blob/master/geocoder/glossary.md#outputSRS	Changed to: https://github.com/bcgov/api-specs/tree/master/geocoder
Install the oc command line tool	available from Redhat	https://docs.openshift.com/container-platform/4.12/web_console/odc-using-web-terminal.html#odc-access-web-terminal_odc-using-web-terminal	Changed to: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/web_console/web-terminal
Set up advanced metrics in Sysdig Monitor	import your application metrics endpoint into Sysdig metrics	https://docs.sysdig.com/en/docs/sysdig-monitor/monitoring-integrations/custom-integrations/collect-prometheus-metrics/migrating-from-promscrape-v1-to-v2/#migrate-using-default-configuration	Changed to: https://docs.sysdig.com/en/docs/sysdig-monitor/integrations/working-with-integrations/custom-integrations/collect-prometheus-metrics/migrating-from-promscrape-v1-to-v2/#migrating-from-promscrape-v1-to-v2
Set up advanced metrics in Sysdig Monitor	Migrate Using Default Configuration	https://docs.sysdig.com/en/docs/sysdig-monitor/monitoring-integrations/custom-integrations/collect-prometheus-metrics/migrating-from-promscrape-v1-to-v2/#migrate-using-default-configuration	Changed to: https://docs.sysdig.com/en/docs/sysdig-monitor/integrations/working-with-integrations/custom-integrations/collect-prometheus-metrics/migrating-from-promscrape-v1-to-v2/#migrate-using-default-configuration
Set up a User Defined Monitoring in OpenShift	Sysdig - Automatically scraping any Kubernetes pods	https://docs.sysdig.com/en/docs/sysdig-monitor/monitoring-integrations/custom-integrations/collect-prometheus-metrics/#agent-compatibility	Changed to: https://docs.sysdig.com/en/docs/sysdig-monitor/integrations/working-with-integrations/custom-integrations/collect-prometheus-metrics/#agent-compatibility
BC Government OpenShift DevOps Security Considerations	https://www2.gov.bc.ca/gov/content/governments/services-for-government/bc-bid-resources/goods-and-services-catalogue/im-it-security-services	https://www2.gov.bc.ca/gov/content/governments/services-for-government/bc-bid-resources/goods-and-services-catalogue/im-it-security-services	Changed to: https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/information-security
External training resources	Building Secure and Reliable Systems	[https://static.googleusercontent.com/media/sre.google/en//static/pdf/building_secure_and_reliable_systems.pdf](https://static.googleusercontent.com/media/sre.google/en/static/pdf/building_secure_and_reliable_systems.pdf)	Changed to: https://sre.google/books/building-secure-reliable-systems/#:~:text=Security%20is%20crucial%20to%20the,systems%20that%20are%20fundamentally%20secure.
Argo CD usage	https://kustomize.io/tutorial	https://kustomize.io/tutorial	Changed to: https://kustomize.io/
About the design system	Joint Working Group (2 times on the page)	https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/digital-delivery/web-property-process/web-property-applications	Changed to: https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/digital-delivery/web-property-process
Deploy an Application	official OpenShift documentation	https://docs.openshift.com/container-platform/latest/applications/application_life_cycle_management/creating-applications-using-cli.html	Changed to: https://docs.openshift.com/container-platform/4.8/applications/creating_applications/creating-applications-using-cli.html
Maintain an application	Check application health after outage	https://docs.developer.gov.bc.ca/maintain-an-application/check-application-health-after-outage/	Changed to: https://docs.developer.gov.bc.ca/check-application-health-after-outage/
Maintain an application	Building secure and reliable systems e-book	https://static.googleusercontent.com/media/sre.google/en//static/pdf/building_secure_and_reliable_systems.pdf	Changed to: https://sre.google/books/building-secure-reliable-systems/

### CODE REVIEW

File: src/docs/build-deploy-and-maintain-apps/deploy-an-application.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ For guidance on resource tuning, refer to this [documentation](https://docs.deve
 
 ### 1. Creating a deployment using OpenShift CLI or Web Console
 
-To deploy an application, you can use the OpenShift CLI (`oc new-app`) or the web console. Both offer the same capabilities and your choice will be dependent on personal preference. More information can be found in the [official OpenShift documentation](https://docs.openshift.com/container-platform/latest/applications/application_life_cycle_management/creating-applications-using-cli.html).
+To deploy an application, you can use the OpenShift CLI (`oc new-app`) or the web console. Both offer the same capabilities and your choice will be dependent on personal preference. More information can be found in the [official OpenShift documentation](https://docs.openshift.com/container-platform/4.8/applications/creating_applications/creating-applications-using-cli.html).
```
Comment (mtspn) [inline src/docs/build-deploy-and-maintain-apps/deploy-an-application.md:100]:
```suggestion
To deploy an application, you can use the OpenShift CLI (`oc new-app`) or the web console. Both offer the same capabilities and your choice will be dependent on personal preference. More information can be found in the [official OpenShift documentation](https://docs.openshift.com/container-platform/4.13/applications/creating_applications/creating-applications-using-cli.html).
```
Suggesting the newer version of this doc.

File: src/docs/training-and-learning/training-external-resources.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ The resources below are from sources outside of the B.C. government. They provid
 ## E-books: 
 [Deploying to OpenShift](https://cloud.redhat.com/hubfs/pdfs/Deploying_to_OpenShift.pdf) Red Hat / O'Reilly Publishing
 
-[Building Secure and Reliable Systems](https://static.googleusercontent.com/media/sre.google/en//static/pdf/building_secure_and_reliable_systems.pdf) by Google / O'Reilly Publishing
+[Building Secure and Reliable Systems](https://sre.google/books/building-secure-reliable-systems/#:~:text=Security%20is%20crucial%20to%20the,systems%20that%20are%20fundamentally%20secure.) by Google / O'Reilly Publishing
```
Comment (mtspn) [inline src/docs/training-and-learning/training-external-resources.md:45]:
```suggestion
[Building Secure and Reliable Systems](https://sre.google/books/building-secure-reliable-systems/#:~:text=Security%20is%20crucial%20to%20the,systems%20that%20are%20fundamentally%20secure) by Google / O'Reilly Publishing
```

**Review by mtspn — CHANGES_REQUESTED** [review]
Looks good Pilar, just some minor changes suggested.

**Review by ShellyXueHan — APPROVED** [review]
LGTM!

**Review by mtspn — APPROVED** [review]
Looks good with these changes, thanks Pilar

**Pull request merged**